broker/src/main/java/io/zeebe/broker/engine/impl/SubscriptionApiCommandMessageHandlerService.java

```diff
@@ -36,16 +36,17 @@ public String getName() {
     return actorName;
   }
 
+  @Override
+  protected void onActorStarting() {
```

Comment:
It seems your IDE orders methods lexicographically.

Reply:
Don't know why that happens. I don't remember changing any settings. I will verify it.